app/src/main/java/com/github/h3lp3rs/h3lp/MainPageActivity.kt

```diff
@@ -311,9 +312,20 @@ class MainPageActivity : AppCompatActivity(), OnRequestPermissionsResultCallback
         navView.setNavigationItemSelectedListener {
             when (it.itemId) {
                 R.id.nav_profile -> goToProfileActivity(findViewById(R.id.button_profile))
-                R.id.nav_home -> findViewById<DrawerLayout>(R.id.drawer_layout).closeDrawer(
-                    GravityCompat.START
-                )
+                R.id.nav_home -> {
+                    findViewById<DrawerLayout>(R.id.drawer_layout).closeDrawer(
+                        GravityCompat.START
+                    )
+                    // Only to deselect the home button
+                    goToActivity(MainPageActivity::class.java)
+                }
```

Comment:
As I said in the comment, it's because the button stays selected. It was the simplest way to fix it and nothing changes visually just the main page being launched again as if you went back to it from another activity.